datasets.py

```diff
@@ -69,7 +68,9 @@ def read_transcript(data_path):
 				self.examples
 			)
 		) if duration_filter else self.examples
-		self.examples = [e for e in self.examples if transcripts.audio_name(e[0]) not in exclude]
+
+		self.examples = [e for e in self.examples
+							if transcripts.audio_name(e[0]) not in exclude] if len(exclude) > 0 else self.examples
```

Comment:
Why is the `if len(exclude) > 0` check needed? `transcripts.audio_name(...)` would work even if `audio_name` field is not present